editor/translations/editor/ca.po

```diff
@@ -3576,7 +3576,7 @@ msgid "Storage"
 msgstr "Emmagatzematge"
 
 msgid "Toggle Autoplay"
-msgstr "Reproducció Automàtica﻿"
```

Comment:
Translation file should be updated on Weblate, not here - https://hosted.weblate.org/translate/godot-engine/godot/ca/?checksum=0da823a1e34b0ea5

Reply:
Ok, I have removed the change and submitted an edit to the translation!